template-parts/flexible-cpts/list-item.php

```diff
@@ -62,7 +62,7 @@
                     }
 
                     if(!empty($term_names)){
-                        $field_value = implode("," , $term_names);
+                        $field_value = implode(", " , $term_names);
```

Comment:
We should use semi-colons as per the other fix
```suggestion
                        $field_value = implode("; " , $term_names);
```